src/frequenz/channels/_broadcast.py

```diff
@@ -277,7 +277,7 @@ def new_receiver(self, *, name: str | None = None, limit: int = 50) -> Receiver[
             limit: Number of messages the receiver can hold in its buffer.
 
         Returns:
-            A Receiver instance attached to the broadcast channel.
+            The new receiver attached to this channel.
```

Comment:
This has to be A.
```suggestion
            A new receiver attached to this channel.

Reply:
In general I agree, but since the docstring already says "Return **a* new receiver...", I thought it might be more clear here to talk about **the**, as it is that same one. But I'm accepting the suggestion, I think both are OK.

In any case, [according to the Google-style we should be able to remove this redundant `Return:` section](https://google.github.io/styleguide/pyguide.html#doc-function-returns) completely, it is just that [this is not supported yet by `pydoclint`](https://github.com/jsh9/pydoclint/issues/110).